src/xopen/__init__.py

```diff
@@ -308,20 +311,30 @@ def __repr__(self):
             self._threads,
         )
 
-    def close(self) -> None:
-        if self.closed:
-            return
-        self.closed = True
+    def _terminate(self):
+        """
+        Check if the process is running and terminate if necessairy.
+        Return the expected error code and the expected error message
```

Comment:
"error code" should be "exit code" (also in a couple of other places below, in particular also in variable names, so `error_code` should be `exit_code`)